packages/core/android/src/main/java/com/datadog/reactnative/DdLogs.kt

```diff
@@ -20,18 +19,28 @@ import com.facebook.react.bridge.ReadableMap
  */
 class DdLogs(reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {
 
-    private val nativeInstance: SDKDdLogs = DdBridge.getDdLogs(reactContext)
-
     override fun getName(): String = "DdLogs"
 
+    private val bridgeLogger: Logger by lazy {
```

Comment:
minor: there is no bridge, so can be called just `logger`. Also maybe in the `setLoggerName` we can use result of `getName`.